synthesis/population/spatial/primary/candidates.py

```diff
@@ -129,14 +141,22 @@ def execute(context):
     )
 
     df_locations = context.stage("synthesis.locations.education")
-    df_education = process(context, "education", random, df_persons,
-        df_education_od, df_locations
-    )
+    if context.config("education_location_source") == 'bpe':
+        df_education = process(context, "education", random, df_persons, df_education_od, df_locations)
+    else :
+        df_education = []
+        for prefix, education_type in EDUCATION_MAPPING.items():
+            df_education.append(
+                process(context, "education_" + prefix, random,
+                    df_persons[df_persons["age"].between( education_type["min_age"],education_type["max_age"])],
+                    df_education_od[df_education_od["age_range"]==prefix],df_locations[df_locations["TYPEQU"].str.startswith(education_type["type_edu"])])
```

Comment:
As said above, I'd like to replace TYPEQU with something more expressive. And in that context it would be good to give it distinct values to avoid testing with `startswith` here.